ramls/schemas/dto/filteringConditionValueCollection.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Filtering condition DTO Schema",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "setsFilteringConditions" : {
+      "description": "list of filtering condition values",
+      "type": "array",
+      "id": "filteringConditionValuesList",
```

Comment:
This is a nitpick but the "List" part at the end of the property id feels superfluous to me.

Reply:
Okay, i'll rename it)